Localization/en-US.hjson

```diff
@@ -0,0 +1,222 @@
+Mods: {
+	VanillaTweaks: {
+		ItemTooltip: {
+			ObsidianArmor: 3% increased ranged critical strike chance
+			SwatHelmet:
+				'''
+				15% increased ranged damage
+				10% increased ranged critical strike chance
+				'''
+			Eskimo: Allows touching cold water without freezing
+			Pharaoh: Increases minion damage by 5%
+			Crimson: 4% increased melee damage
+		}
+
+		MiscellaniaTooltip: {
+			SwatHelmet:
+				'''
+				15% increased ranged damage
+				10% increased ranged critical strike chance
+				Provides a set bonus when worn with the Reinforced Vest
+				'''
+		}
+
+		ArmorSet: {
+			Obsidian: 10% increased movement speed
+			Gladiator: 15% increased critical strike chance
+			Swat:
+				'''
+				Reduces damage taken by 25%
+				20% increased ranged damage and chance not to consume ammo
+				'''
+			Eskimo:
+				'''
+				4 defense
+				Provides warmth and immunity to chill, frostburn and freezing effects
+				'''
+			Viking: 5% increased damage
+			Cactus: Attackers also take damage
+			Pharaoh: Increases your max number of minions
+		}
```

Comment:
Maybe we could remove unused text?

Reply:
In that case, we could leave the `.lang.legacy` files as-is, so they can be useful for storing history, I guess.